docs/integration-testing.md

```diff
@@ -53,7 +53,7 @@ any residue before and after the test.
 ## Running them locally
 
 The JUnit runner won't pick up integration tests unless the `RUN_INTEGRATION_TEST` environment variable is set to `true`
-. Also, don't forget to define any credentials that are needed.
+. Also, don't forget to define any credentials that are needed and if you need to run specific integration tests then it can be achieved by passing the `includeTags` parameter to the `gradlew` command e.g. `./gradlew check -DincludeTags="azure-cosmos-db-integration-test"`.
```

Comment:
```suggestion
. Also, don't forget to define any credentials that are needed and if you need to run specific integration tests then it can be achieved by passing the `includeTags` parameter to the `gradlew` command:

 For Example:
 ```script
  `./gradlew check -DincludeTags="azure-cosmos-db-integration-test"`.
  ```
```

Reply:
Above is more of a suggestion to break out the command to make it more readable

Reply:
Thanks, updated the doc as per suggestion.